src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -84,6 +85,7 @@ start_metadata_sync_to_node(PG_FUNCTION_ARGS)
 	List *createMetadataCommandList = NIL;
 
 	EnsureSchemaNode();
+	EnsureAllNodesActivated();
```

Comment:
Well, that's just to simplify the story. I didn't want users to `sync` metadata which is not activated. Also, to avoid corner cases that could arise due to the order of `sync` - `activate` nodes in the future. (Because we're considering to send data types, schemas etc on `activate` in the near future)